blueprints/ember-electron/index.js

```diff
@@ -96,18 +96,13 @@ module.exports = class EmberElectronBlueprint extends Blueprint {
         doc.install.push('__xvfb__');
       }
 
-      if (!doc.install.find(entry => entry.toLowerCase().includes('xvfb'))) {
-        doc.install.push('__export_display__');
-        doc.install.push('__xvfb__');
-      }
-
       // also, yawn quotes strings with certain characters in them even though
       // it isn't necessary, and it makes it harder to read. So we add
       // placeholders that won't be quoted and replace them in the output string
       yawn.json = doc;
       let output = yawn.yaml;
-      output = output.replace('__yarn__install__', `cd electron-app && yarn`);
-      output = output.replace('__npm__install__', `cd electron-app && npm install`);
```

Comment:
I had to stare at this diff for a minute to figure out what changed. Nice catch with the extra `_`